cmd/photobak/main.go

```diff
@@ -47,6 +48,13 @@ type daemon struct {
 }
 
 func startDaemon(interval time.Duration) {
+	if runtime.GOOS != "windows" {
+		// The default behaviour on SIGPIPE is to silently terminate the program which breaks clean shutdown, so ignore
+		// it because every program should check write() return code instead of crashing if some file descriptor became
+		// unavailable for writing.
+		signal.Notify(make(chan os.Signal), syscall.SIGPIPE)
```

Comment:
Yes, of course, I've checked that `cmd/photobak/build.bash` compiles without any error. It looks like the constant is defined here - https://github.com/golang/go/blob/master/src/syscall/ztypes_windows.go#L62 I believe it's either Posix level of compatibility for Windows or just a workaround to make conditional compilation less cumbersome.